Jellyfin.Plugin.KodiSyncQueue/API/KodiSyncQueueController.cs

```diff
@@ -195,26 +200,29 @@ private List<string> GetAddedOrUpdatedItems(User user, IEnumerable<Guid> ids)
 
             _logger.LogDebug("Starting PopulateLibraryInfo...");
 
-            var info = new SyncUpdateInfo();
-
             var userDt = DateTime.Parse(lastRequestedDt, CultureInfo.CurrentCulture, DateTimeStyles.AssumeUniversal);
             var dtl = (long)userDt.ToUniversalTime().Subtract(new DateTime(1970, 1, 1, 0, 0, 0, 0, DateTimeKind.Utc)).TotalSeconds;
             var user = _userManager.GetUserById(Guid.Parse(userId));
 
-            var itemsAdded = Plugin.Instance.DbRepo.GetItems(dtl, ItemStatus.Added, filters);
-            var itemsRemoved = Plugin.Instance.DbRepo.GetItems(dtl, ItemStatus.Removed, filters);
-            var itemsUpdated = Plugin.Instance.DbRepo.GetItems(dtl, ItemStatus.Updated, filters);
-            var userDataChanged = Plugin.Instance.DbRepo.GetUserInfos(dtl, userId, filters);
-
-            info.ItemsAdded = GetAddedOrUpdatedItems(user, itemsAdded);
-            info.ItemsRemoved = itemsRemoved.Select(id => id.ToString("N")).ToList();
-            info.ItemsUpdated = GetAddedOrUpdatedItems(user, itemsUpdated);
-            info.UserDataChanged = userDataChanged.Select(i => JsonSerializer.Deserialize<UserItemDataDto>(i.JsonData, JsonDefaults.GetOptions())).ToList();
-
-            _logger.LogInformation("Added: {AddedCount}, Removed: {RemovedCount}, Updated: {UpdatedCount}, Changed User Data: {ChangedUserDataCount}",
-                info.ItemsAdded.Count, info.ItemsRemoved.Count, info.ItemsUpdated.Count, info.UserDataChanged.Count);
+            var itemsAdded = KodiSyncQueuePlugin.Instance.DbRepo.GetItems(dtl, ItemStatus.Added, filters);
+            var itemsRemoved = KodiSyncQueuePlugin.Instance.DbRepo.GetItems(dtl, ItemStatus.Removed, filters);
+            var itemsUpdated = KodiSyncQueuePlugin.Instance.DbRepo.GetItems(dtl, ItemStatus.Updated, filters);
+            var userDataChanged = KodiSyncQueuePlugin.Instance.DbRepo.GetUserInfos(dtl, userId, filters);
+
+            var info = new SyncUpdateInfo(
+                GetAddedOrUpdatedItems(user, itemsAdded),
+                itemsRemoved.Select(id => id.ToString("N", CultureInfo.InvariantCulture)).ToList(),
+                GetAddedOrUpdatedItems(user, itemsUpdated),
+                userDataChanged.Select(i => JsonSerializer.Deserialize<UserItemDataDto>(i.JsonData, JsonDefaults.GetOptions())).ToList());
```

Comment:
`JsonDefaults.GetOptions()` should be put in a private readonly field (consistent with server and faster)